source/_integrations/xiaomi_miio.markdown

```diff
@@ -491,6 +493,29 @@ Buzzer                  | Turn on/off the buzzer
 Child Lock              | Turn on/off the child lock
 LED                     | Turn on/off the LED
 
+### Air Fresh A1 (dmaker.airfresh.a1)
+
+- Power (on, off)
+- Operation modes (Auto, Sleep, Favorite)
+- Sensor entities
+
+Sensor                          | Description                                                    | Enabled by default
+------------------------------- | -------------------------------------------------------------- | -----------------------
+Carbon Dioxide                  | The current carbon dioxide measured in ppm                     | True
+Dust filter life remaining      | The remaining life of the filter                               | True
+Dust filter life remaining days | The remaining life of the filter in day                        | True
+PM2.5                           | The current particulate matter 2.5 measured                    | True
+Temperature                     | The current outside temperature measured                       | True
+Control Speed                   | The current motor speed measured in rpm                        | True
+Favorite Speed                  | The favorite motor speed measured in rpm                       | True
```

Comment:
There is no need to list they are all enabled by default, if they are ALL enabled by default. Please remove that column.

Reply:
done